llpc/builder/llpcBuilder.h

```diff
@@ -159,8 +160,8 @@ class Builder : public BuilderBase
     // Get the BuilderContext
     BuilderContext* GetBuilderContext() const { return m_pBuilderContext; }
 
-    // Set the current shader stage, clamp shader stage to the ShaderStageCompute
-    void SetShaderStage(ShaderStage stage) { m_shaderStage = (stage > ShaderStageCompute) ? ShaderStageCompute : stage; }
+    // Set the current shader stage, clamp shader stage to the ShaderTypeCompute
+    void SetShaderType(ShaderType stage) { m_shaderStage = (stage > ShaderTypeCompute) ? ShaderTypeCompute : stage; }
```

Comment:
Better to make this change in middle-end: m_shaderStage -> m_shaderType for all affected parts.

Reply:
Not applicable now.